### PR TITLE
Show cfd expiry in UI

### PR DESCRIPTION
**Pull request by bonomat**
Looks so pretty :) 

image

### CODE REVIEW

File: daemon/src/model/cfd.rs [hunk 1]
```diff
@@ -689,6 +689,10 @@ impl Cfd {
         (self.order.term * Cfd::REFUND_THRESHOLD).as_blocks().ceil() as u32
     }
 
+    pub fn expiry_timestamp(&self) -> i64 {
+        self.order.oracle_event_id.as_unix_timestamp()
```
Comment (bonomat) [inline daemon/src/model/cfd.rs]: Ideally this is stored as a field on the CFD and we don't have to parse it from the oracle event id but I didn't want to change the data model.

File: daemon/src/model.rs [hunk 2]
```diff
@@ -207,6 +207,10 @@ impl BitMexPriceEventId {
             .join(&self.to_string())
             .expect("Event id can be joined")
     }
+
+    pub fn as_unix_timestamp(&self) -> i64 {
+        self.timestamp.unix_timestamp()
+    }
```
Comment (thomaseizinger) [inline daemon/src/model.rs]: I think I'd prefer if we expose the strongly-typed timestamp (`OffsetDatetime`) instead of the unix timestamp.

**Review by thomaseizinger — APPROVED** [review]

File: daemon/src/model.rs (hunk 2 above)
Comment (bonomat) [reply daemon/src/model.rs]: Force pushed, please check again :)

**Review by thomaseizinger — APPROVED** [review]
Nice use of the `serde` module from `time`!

**Pull request merged**